dashboard/app/models/user.rb

```diff
@@ -494,6 +494,7 @@ def user_level_for(script_level, level)
 
   def user_level_locked?(script_level, level)
     return false unless script_level.stage.lockable?
+    return false if authorized_teacher?
```

Comment:
Shouldn't this fall through to `user_level.locked?` below if there is a UserLevel? https://github.com/code-dot-org/code-dot-org/pull/10198/files#diff-dc570f6aec74faac27d58b35ea145b48R107